__TESTS__/helper/i18n.tsx

```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I think we can [mock i18next](https://dev.to/mehmetnyarar/how-to-test-next-i18next-1nh4) in jest instead of using a helper here

Reply:
I believe that using MOCK is a good idea.
However, some tests use element names, etc., which can be problematic if they are provided in i18n.
In that case, by render with this helper function, the English string will be inserted in the original page.

Reply:
You can also use a mock-up if that is not needed. That is going to have a positive impact on performance.
This also makes it difficult to write the mock in the setup file...

Reply:
I will explore this a bit tonight and see what I can come up with.

Reply:
I see what you mean now, I think your implementation is better, we'll leave this as is.